### ISSUE TITLE
Replace expansion panels with USWDS accordion

### ISSUE THREAD

**Issue by greensomes** [DOI-ONRR/onrr.gov-site#3199]
This issue is related to the "This BUTTON has an aria-controls attribute value of 'panel-content-0'" violations in issue #2706. The replacement will allow us to resolve those violations since USWDS renders the hidden content referred to by the aria-controls attribute.

**Comment by eelzi-ONRR** [comment]
I think I looked at this earlier in the week? If so it looked good!
